### ISSUE TITLE
Using LinqToDb in all queries with EF Core Context

### ISSUE THREAD

**Issue by MohrWizard** [linq2db/linq2db.EntityFrameworkCore#290]
Reading to the docs I successfully managed to have LinqToDb to create  sql queries using ToLinqToDb() and ToListAsyncLinqToDb() ( and all other async extension methods) but I want to know if it is  possible to tell Ef core to always use LinqToDb queries by default. For example I can tell Ef core to split a specific query with AsSplitQuery() but I can also make all queries split by default with UseQuerySplittingBehavior(QuerySplittingBehavior.SplitQuery). Does LinqToDb have something equivalent? If not can you consider adding such a feature?

**Comment by sdanyliv** [comment]
1. Not sure it is possible to replace all EF Core queries by default. I have tried it, but it needs a lot of efforts and implementing required interfaces.

3. Linq2db always use Split queries. We have not implemented single query approach.

**Comment by MohrWizard** [comment]
> 1. Not sure it is possible to replace all EF Core queries by default. I have tried it, but it needs a lot of efforts and implementing required interfaces.
> 2. Linq2db always use Split queries. We have not implemented single query approach.
I am maintaing a wpf application based on ef core 3.1 and .Net framework 4.8 so your library is a life saver for me

**Comment by MohrWizard** [comment]
Thank you so much for the fast reply. Your library is a life saver for me. I am maintaining a .net framework application and when I updated Ef core from 2.0 to 3.1 I got the wonderful surprise of Cartesian explosion. I cannot upgrade to ef core 5.0 since it requires .net standard 2.1. I wish I could solve all this mess with a single line of code but using some regex to change all ToListAsync and FirstOrDefaultAsync to the LinqToDb version doesn't seem a big effort like changing all my queries to use explicit loading instead of eager loading.
